lib/Yam/ControlCenterPage.pm

```diff
@@ -0,0 +1,370 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles Control Center
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::ControlCenterPage;
+use parent 'Yam::PageBase';
+use strict;
+use warnings;
+use testapi;    # needed because of bsc1206966
+
+sub init {
+    my $self = shift;
+    $self->{lbl_yast_cc} = $self->{app}->label({label => 'YaST Control Center'});
+    $self->{slb_groups} = $self->{app}->selectionbox({id => 'groups'});
+    $self->{slb_progs} = $self->{app}->selectionbox({id => 'progs'});
+    $self->{btn_help} = $self->{app}->button({id => 'help'});
+    $self->{btn_run} = $self->{app}->button({id => 'run'});
+    $self->{btn_quit} = $self->{app}->button({id => 'quit'});
+    return $self;
+}
+
+sub get_control_center_page {
+    my ($self) = @_;
+    die 'YaST Control Center is not displayed' unless $self->{btn_help}->exist();
+    return $self;
+}
+
+# workarounds
+
+sub apply_workaround_bsc1206966 {
+    my ($self) = @_;
+    record_soft_failure('bsc#1206966 - Item selection with REST API does not update contents');
+    my $currentgroup = $self->{slb_groups}->selected_items();
+    send_key 'shift-tab';
+    if ($currentgroup eq 'Software') {
+        send_key 'down';
+        send_key 'up';
+    }
+    else {
+        send_key 'up';
+        send_key 'down';
+    }
+}
+
+# private methods collection
+
+sub _select_software {
+    my ($self) = @_;
+    $self->{slb_groups}->select('Software');
+    apply_workaround_bsc1206966();
```

Comment:
you are forgetting to return the `$self` for each single call as we still stay in the screen.
Chaining calls is quite nice, but I'm not sure if the libyui-rest-api logs are good enough, the problem was that the failing line was confusing in an error (but after stack trace was introduced might not be an issue, we need to revisit that).
In case of doubt we could follow the same design, and return the `$self`, it might be useful to trigger help menu and do it as one-liner :)